### PR TITLE
Typo fixes throughout docs and namespace

### LINKED ISSUE #36: Fix discovered typos
- Attribute is called Atrribute in several places
- Some signals in the collision module use 'seperated' instead of 'separated'

### PR DESCRIPTION
**Pull request by 20milliliter**
Implements most of the changes raised within my review of #34.

Fixes #36.

### CODE REVIEW

**Comment by Pyxus** [comment]
Thank you for your interest in contributing! I apologize if your efforts go wasted but I had actually already implemented the majority of your corrections prior to this pull request on the this branch: https://github.com/Pyxus/fray/tree/fix-documentation-errors. I also went ahead and made an effort to improve the flow of many of my sentences and corrected other grammatical errors I noticed while doing so. I'll be reviewing your PR to see if there is anything I had missed but otherwise this will be superseded by my upcoming PR.

To be honest I'm a bit inexperienced when it comes to managing an open source project like this. I think after the alpha or beta release (which should be close) I'll look into implemented some better git practices into my workflow and being more transparent with what is being worked on among other things. Just to minimize any potential friction as more people become interested in the project.

File: addons/fray/assets/icons/hit_attributes.svg.import
Comment (Pyxus) [inline addons/fray/assets/icons/hit_attributes.svg.import:1]: I'll be addressing these attribute typos in a different PR once the documentation fixes are merged. This resulted from a typo in a mass rename I did a while ago. Simple fix but it touches multiple files in the project beyond the documentation.

**Review by Pyxus — APPROVED** [review]
On second thought since this PR also fixes the typos outside of the documentation that I wanted to resolve i'll be merging. Not sure if this is the best approach but I believe this way I can cherry pick those changes into my other branch.

**Comment by 20milliliter** [comment]
> To be honest I'm a bit inexperienced when it comes to managing an open source project like this.

Not a problem, I'm in the same boat myself.
My bad for not checking branches, can't believe I'd forget something like that.

**Pull request merged**